syntax = "proto3";

package main;


message ItemRequest {
  string group = 1;

  string item = 2;
}

message ItemResponse {
  bool success = 1;
}

message ItemsRequest {
  string group = 1;
}

message ItemsResponse {
  
  repeated string items = 1;
}

service ListService {
  rpc AddItem(ItemRequest) returns (ItemResponse); 

  rpc GetItems(ItemsRequest) returns (ItemsResponse); 
}
